Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -61,6 +64,14 @@ namespace Babylon
             bool Dirty{};
         } m_bgfxState{};
 
+        struct
+        {
+            std::mutex Mutex{};
```

Comment:
where is this used?

Reply:
This struct reperesents our "target resolution" of the application. This mutex is used so we can retrieve and change this resolution in a thread-safe way.

Reply:
I don't see any references to this variable. Am I missing something?

Reply:
Oh, you added it just now.